recipes/vphaser2/meta.yaml

```diff
@@ -1,6 +1,7 @@
 about:
     home: 'https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/v-phaser-2'
-    license: 'https://www.broadinstitute.org/node/4962'
+    license: 'single user license for academic non-commercial research purposes only'
+    license_file: 'https://github.com/broadinstitute/v-phaser2/blob/master/V-Phaser-2.0/license/V_Phaser_2_License_v20130317.pdf'
```

Comment:
This should point to a file in the tarball.

Reply:
ok, I saw several other packages using references to github, so did the same, gonna fix this